web-ui/src/main/resources/catalog/components/metadatamanager/EditorService.js

```diff
@@ -184,16 +184,6 @@
             );
           return defer.promise;
         },
-        /**
-         * Save the metadata record currently in editing session.
-         *
-         * If refreshForm is true, then will also update the current form.
-         * This is required while switching tab for example. Update the tab
-         * value in the form and trigger save to update the view.
-         */
-        save: function (refreshForm, silent, terminate) {
-          save(refreshForm, silent, terminate, false, false);
-        },
```

Comment:
Is this change correct?

Reply:
we have the same function with one more arg below  `save: function (refreshForm, silent, terminate, submit, approve) {` and approve will be undefined by default so should be fine.